app/scripts/modules/amazon/src/domain/IAmazonLoadBalancerSourceData.ts

```diff
@@ -0,0 +1,127 @@
+import { ILoadBalancerSourceData } from '@spinnaker/core';
+
+export interface IAmazonLoadBalancerServerGroupSourceData {
+  detachedInstances: string[];
+  instances: IAmazonLoadBalancerInstanceSourceData[];
+  isDisabled: boolean;
+  name: string;
+  region: string;
+}
+
+export interface IAmazonInstanceHealthSourceData {
+  type: string;
+  state: 'InService' | 'OutOfService' | 'Unknown' | 'healthy';
```

Comment:
It should not, but it is actually a different health state, so I'll extract this a little further.